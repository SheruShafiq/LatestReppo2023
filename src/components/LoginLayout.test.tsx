import React from "react";
import { render } from "@testing-library/react";
import matchers from "@testing-library/jest-dom";
import { debug } from "vitest-preview";
import LoginLayout from "../components/LoginLayout";
expect.extend(matchers);

describe("LoginLayout", () => {
  it("renders its children correctly", async () => {
    const children = <div data-testid="child-element">Child</div>;
    const { getByTestId } = await render(<LoginLayout>{children}</LoginLayout>);

    const childElement = getByTestId("child-element");
    expect(childElement).toBeInTheDocument();
    expect(screen).toMatchSnapshot();
  });
});
