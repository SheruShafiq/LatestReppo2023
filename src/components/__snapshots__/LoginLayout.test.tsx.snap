// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`LoginLayout > renders its children correctly 1`] = `Screen {}`;
